WordPress/src/main/java/org/wordpress/android/ui/notifications/utils/NotificationsUtilsWrapper.kt

```diff
@@ -0,0 +1,50 @@
+package org.wordpress.android.ui.notifications.utils
+
+import android.text.Spannable
+import android.widget.TextView
+import org.json.JSONObject
+import org.wordpress.android.fluxc.tools.FormattableContent
+import org.wordpress.android.fluxc.tools.FormattableContentMapper
+import org.wordpress.android.ui.notifications.blocks.NoteBlock
+import javax.inject.Inject
+import javax.inject.Singleton
+
+/**
+ * Injectable wrapper around NotificationsUtils.
+ *
+ * NotificationsUtils interface is consisted of static methods, which make the client code difficult to test/mock.
+ * Main purpose of this wrapper is to make testing easier.
+ *
+ */
+@Singleton
+class NotificationsUtilsWrapper @Inject constructor() {
+    @Inject protected lateinit var formattableContentMapper: FormattableContentMapper
```

Comment:
I think you should move this into the constructor. That way it could be a `val`

Reply:
👍